dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23CDIInjectionTests.java

```diff
@@ -218,6 +221,7 @@ public void testBeanInjection() throws Exception {
      * Does some simple verifications of the 4 scopes and instances ( through hashcode) are what is expected for multiple requests.
      */
     @Test
+    @SkipForRepeat(SkipForRepeat.EE10_FEATURES)
     public void testViewScopeInjections() throws Exception {
```

Comment:
I'm can't run my jsf 2.3 fat locally for some reason, but why was this one skipped?

Reply:
If I have it skipped it is b/c it was failing and I've added it to: https://github.com/OpenLiberty/open-liberty/issues/22368 for further investigation.